json-endpoints/jetpack/class.jetpack-json-api-plugins-install-endpoint.php

```diff
@@ -38,6 +38,12 @@ protected function install() {
 		}
 
 		if ( ! $this->bulk && isset( $error ) ) {
+
+			if ( $error_code === 'download_failed' ) {
+				// For backwards compatibility: versions prior to 3.9 would return no_package instead of download_failed.
+				$error_code = 'no_package';
```

Comment:
Was this documented before? If not, I think we should return the more accurate error code.